lib/mojito.js

```diff
@@ -108,7 +108,7 @@ Mojito = (function () {
                 if (previewRecipe != null) {
                     this.log('Forcing test [' + this.options.name + '][' + this.options.id + '] into recipe [' + previewRecipe.name + '][' + recipeId + ']');
 
-                    this.setInTest(1);
+                    this.setInTest('1');
```

Comment:
Were there type problems related to Utils.keys/arrayIndexOf, that this addresses?

Reply:
Yes, but this is mainly for the changes: `==` to `===`.

Reply:
Cool - thought so.